br/pkg/version/version.go

```diff
@@ -241,11 +242,22 @@ func NormalizeBackupVersion(version string) *semver.Version {
 }
 
 // FetchVersion gets the version information from the database server
+//
+// NOTE: the executed query will be:
+// - `select tidb_version()` if target db is tidb
+// - `select version()` if target db is not tidb
 func FetchVersion(ctx context.Context, db utils.QueryExecutor) (string, error) {
 	var versionInfo string
+	const queryTiDB = "SELECT tidb_version();"
```

Comment:
When is this supported? Will we have a compatible issue?

Reply:
iirc we have the `tidb_version()` function since the very beginning...

Reply:
gotta